app/scripts/directives/notifications/notificationDrawerWrapper.js

```diff
@@ -246,7 +249,7 @@
       angular.extend(drawer, {
         drawerHidden: true,
         allowExpand: true,
```

Comment:
Ah so you would prefer it would remember this state

Reply:
The idea is if you prefer it open to see descriptions, you don't have to click the little expand button every time. And we tend to remember other collapsed / expanded states like overview list rows and nav.